router/src/main/resources/Dockerfile

```diff
@@ -39,4 +57,16 @@ ENV ENFORCER_CA_CERT_PATH=/home/wso2/security/truststore/mg.pem
 
 ENV TRAILING_ARGS=
 
+RUN mkdir -p /etc/envoy
+COPY --from=0 /usr/local/bin /usr/local/bin
+COPY --from=0 /etc/envoy/envoy.yaml /etc/envoy/envoy.yaml
```

Comment:
We do not need the copy statement here I guess. Since we are copying from build machine.

Reply:
removed https://github.com/wso2/product-microgateway/pull/1859/commits/3641c04458402774fa0815bad6af427ce37823fd